testing/glide/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Aaron Hurt <ahurt@ena.com>
+# Maintainer: Aaron Hurt <ahurt@ena.com>
+pkgname="glide"
+pkgver="0.10.2"
+pkgrel=0
+pkgdesc="Vendor Package Management for Golang"
+url="https://github.com/Masterminds/glide"
+arch="all"
+license="MIT"
+depends=""
+makedepends="go"
+install=""
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/Masterminds/glide/archive/$pkgver.tar.gz"
+builddir="$srcdir/go/src/github.com/Masterminds/glide"
+options="!strip"
```

Comment:
Is there any reason not to strip the binary? I always thought it is possible without causing issues.